src/styles/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(/images/star-active.svg);
```

Comment:
Use relative path for the background-image to avoid issues with different folder structures. Replace url('/images/star-active.svg') with url('../images/star-active.svg') 🖼️